Dockerfile

```diff
@@ -94,6 +94,7 @@ RUN /nnunet_env/bin/pip install git+https://github.com/MIC-DKFZ/nnUNet.git@nnune
 
 ENV nnUNet_raw_data_base="/tmp/nnUNet_raw_data_base"
 ENV nnUNet_preprocessed="/tmp/nnUNet_preprocessed"
+ENV CUDA_VISIBLE_DEVICES="0"
```

Comment:
@hasan7n might be able to provide a better answer, but from my understanding, when running a Docker image passing the specific GPU(s) to use, Docker reassigns the ids from 0 to n. So for example passing
`docker run ... --gpus "device=2"`, it will be identified within the Docker image with `id=0`.

@hasan7n let me know if I understood correctly. If you can also put a link to that info as a comment for this line in the docker image that would be great! I'll see if I can find it myself

Reply:
Completely true @aristizabal95 !

@sarthakpati that's of course however something to consider if we want to allow multiple-gpu execution. we can revisit this when it's needed.